packages/3id-did-resolver/README.md

```diff
@@ -23,7 +23,6 @@ import { Resolver } from 'did-resolver'
 
 // You need an instance of Ceramic to call getResolver.
 // This can be either @ceramicnetwork/core or @ceramicnetwork/http-client.
-// You can also set an address for your own ethr-did-registry contract
```

Comment:
can we not do this anymore?  I thought we did still support ethr-did.

@oed

Reply:
oh I could definitely be wrong about this. It looked like an old copy-paste error to me (it's identical text to what's on the ethr-did-resolver README). if it's intentional maybe the example could be expanded to clarify it?